packages/uikit-react-native-foundation/src/components/TextInput/index.tsx

```diff
@@ -38,8 +42,11 @@ const TextInput = React.forwardRef<RNTextInput, Props>(function TextInput(
 
 const styles = createStyleSheet({
   input: {
-    paddingVertical: 8,
-    paddingHorizontal: 16,
```

Comment:
paddingVertical / paddingHorizontal is not working due to bug in React-Native
https://github.com/facebook/react-native/pull/33564